Mburia/count_zeros.py

```diff
@@ -0,0 +1,26 @@
+# Dependency
+import numpy as np
```

Comment:
Could you try do it without any library? Just loops, etc.

Reply:
Done

Reply:
Done